hack/agent-cli/README.md

```diff
@@ -32,9 +32,11 @@ cc-agent
 
 From a different shell on the host (as root):
 ```
-# ./agent-cli run --ctl=/tmp/hyper.sock --tty=/tmp/tty.sock
+# ./agent-cli run --ctl=/tmp/hyper.sock --tty=/tmp/tty.sock --wait-for-ready
 ```
 
+In case the agent-cli is terminated or disconnected from `CTL` and `TTY` sockets, and the agent is still listening on those sockets, you should not use the flag `--wait-for-ready` since the agent will not send any `READY` message.
```

Comment:
I suggest a note:

note:
Do not use the `--wait-for-ready` flag when the agent-cli is terminated or disconnected from the `CTL` and `TTY` sockets and the agent is still listening on those sockets. In this case, the agent will not send a `READY` message.

Reply:
Thanks for the comment, this is better. I have updated the code.